source/frontend/src/features/leases/detail/LeasePages/payment/table/periods/PaymentPeriodsView.tsx

```diff
@@ -28,7 +28,9 @@ export interface IPeriodsFormProps {
   formikRef: React.RefObject<FormikProps<LeaseFormModel>>;
 }
 
-export const PeriodsForm: React.FunctionComponent<React.PropsWithChildren<IPeriodsFormProps>> = ({
+export const PeriodPaymentsView: React.FunctionComponent<
```

Comment:
nit: please rename file to match component name: `PaymentPeriodsView.tsx` -> `PeriodPaymentsView.tsx`

Reply:
> nit: please rename file to match component name: `PaymentPeriodsView.tsx` -> `PeriodPaymentsView.tsx`

will do in follow up PR